src/Service.GraphQLBuilder/CustomScalars/SingleType.cs

```diff
@@ -13,7 +12,7 @@ namespace Azure.DataApiBuilder.Service.GraphQLBuilder.CustomScalars
     /// </summary>
     public class SingleType : FloatTypeBase<float>
     {
-        public static readonly NameString TypeName = new("Single");
```

Comment:
We removed NameString in Hot Chocolate 13 as this semantic string object just caused a lot of allocation.